package.json

```diff
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/scality/S3#readme",
   "dependencies": {
-    "arsenal": "github:scality/Arsenal#3ed66c5",
+    "arsenal": "github:scality/Arsenal#5b6b6ee",
```

Comment:
Updated this one here as well, based on the latest bert-e hash on here: https://github.com/scality/Arsenal/pull/971 is this the right approach?